test/MinimalApi6.0-Swashbuckle6/Program.cs

```diff
@@ -1,9 +1,13 @@
 using Microsoft.OpenApi.Models;
 using Swashbuckle.AspNetCore.Filters;
+using System.Reflection;
 
 var builder = WebApplication.CreateBuilder(args);
 
+
 // Add services to the container.
+builder.Services.AddControllers();
```

Comment:
As per its name, this test project is just for testing Minimal API and not controllers, so it shouldn't have this.